src/entities/models/dao.py

```diff
@@ -61,14 +61,23 @@ def __str__(self):
         return f"ID: {self.id},Filing ID: {self.filing}, Task: {self.task}, User: {self.user}, state: {self.state}, Timestamp: {self.change_timestamp}"
 
 
+class ContactInfoDAO(Base):
+    __tablename__ = "contact_info"
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    filing: Mapped[int] = mapped_column(ForeignKey("filing.id"))
+    first_name: Mapped[str]
+    last_name: Mapped[str]
+    email: Mapped[str]
+    phone: Mapped[str]
+
+
 class FilingDAO(Base):
```

Comment:
contact_info isn't a string anymore, but we need it to be a relationship, refer to how the `tasks` is populated.